handlers.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/rolandovlz/goauth/models"
```

Comment:
This import will fail in any other goPath besides your own, no? Is there a way to import another package in the gopath in a "generic" way?